docs/parse_argparse.py

```diff
@@ -114,10 +114,14 @@ def get_uasge(argparse_lines, fpath):
     proc = subprocess.run(cmd, stdout=subprocess.PIPE,
                           stderr=subprocess.DEVNULL, shell=True, text=True)
     # checks if the command was run properly else raises error
+    # proc.check_returncode()
     try:
         proc.check_returncode()
-    except:
-        print(f"Error while parsing argparse block of {fpath}")
+    except subprocess.CalledProcessError as err:
+        print(err)
+        print(
+            f"Error while running --help on these argparse lines \n {argparse_lines}")
+        raise err
```

Comment:
When the error is raised, this information will be emitted to the terminal. printing(err) as well as raising it is redundant.

```suggestion
        print(
            f"Error while running --help on these argparse lines \n {argparse_lines}")
        raise err
```